MIGRATING.md

```diff
@@ -2,6 +2,24 @@
 
 This guide explains what is needed to upgrade contracts when migrating over major releases of `sylvia`. Note that you can also view the [complete CHANGELOG](https://github.com/CosmWasm/sylvia/blob/main/CHANGELOG.md) to understand the differences.
 
+## 0.5.0 -> Unreleased
+
+- InstantiateCtx and ExecCtx could be previosly used interchangeably. They are currently separate
+  types. The same applies to ReplyCtx/MigrateCtx pair.
+
+```
+error[E0308]: mismatched types
+  --> contracts/cw1-subkeys/src/contract.rs:56:49
+   |
+56 |         let result = self.whitelist.instantiate(ctx.branch(), admins, mutable)?;
+   |                                     ----------- ^^^^^^^^^^^^ expected `ExecCtx<'_>`, found `InstantiateCtx<'_>`
+   |                                     |
+   |                                     arguments to this method are incorrect
+   |
+   = note: expected struct `ExecCtx<'_>`
+              found struct `InstantiateCtx<'_>`
+```
+
```

Comment:
Too long of an entry for changelog. The only description is enough.

Reply:
This is MIGRATING, but applied some changes as discussed offline